homeassistant/components/deconz/strings.json

```diff
@@ -11,11 +11,14 @@
         "data": {
           "host": "[%key:common::config_flow::data::host%]",
           "port": "[%key:common::config_flow::data::port%]"
+        },
+        "data_description": {
+          "host": "The hostname or IP address of your deCONZ device."
```

Comment:
```suggestion
          "host": "The hostname or IP address of your deCONZ host."
```

Reply:
Thank you for your help, @Kane610. Much appreciated.

Reply:
Thank you @c0ffeeca7